ui/src/components/error/NavWrapperErrorFallback.tsx

```diff
@@ -0,0 +1,28 @@
+import { Trans } from '@lingui/macro';
+import './index.css';
+
+interface ErrorFallbackProps {
+  error: Error;
+  resetErrorBoundary: () => void;
+}
+
+function NavWrapperErrorFallBack({ error }: ErrorFallbackProps) {
+  return (
+    <div className="error-container">
+      <div className="error-title">
+        <Trans>Something went wrong</Trans>
+      </div>
```

Comment:
Should be an `h2`, at least for now.

Reply:
Removed this component to re-use AppErrorFallback for all error boundaries